ansible/roles/schulcloud-server-tspsync/tasks/main.yml

```diff
@@ -81,3 +81,33 @@
     when: not WITH_TSP
     tags:
       - cronjob
+
+  - name: API TSP Sync CronJob
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: api-tsp-sync-cronjob.yml.j2
+    when: WITH_TSP
+    tags:
+      - cronjob
+
+  - name: remove API TSP Sync CronJob
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      state: absent
+      api_version: batch/v1
+      kind: CronJob
+      namespace: "{{ NAMESPACE }}"
+      name: api-tsp-sync-cronjob
+    when: not WITH_TSP
```

Comment:
Use an different condition as the `WITH_TSP` other wise the new tsp sync cronjob is add right know to production